.circleci/config.yml

```diff
@@ -3,10 +3,104 @@ version: 2.1
 orbs:
   silta: silta/silta@0.1
 
+images:
+  - &node
+    image: circleci/node:12.16.1-browsers
```

Comment:
node 16 LTS is out, I'd use that!

Reply:
Also I think circleci have deprecated these images in favour of convenience images (`cimg`). Here are the node ones:

https://circleci.com/developer/images/image/cimg/node